modules/_unstable/README.md

```diff
@@ -0,0 +1,44 @@
+# Canvas Kit Unstable
+
+This module is a sandbox for work in progress components.
+
+Since we have a rather rigorous process for getting components in at a production level, it can be
+valuable to make them available earlier for teams who need them during development. Many of our
+components are contributions from internal product teams. While these components can suit the
+contributing product's use case, it takes time to iterate on the API/functionality as we evaluate
+them in a more global context. The Unstable module allows us to do that as needed.
+
+## Breaking Changes
+
+Due to the nature of this bundle of components, we will be continuously iterating, likely making
+breaking changes. The goal of this module is to work unconstrained by our design system's semantic
+versioning. This means that **breaking changes can be deployed to
+`@workday/canvas-kit-react-unstable` or `@workday/canvas-kit-css-unstable` at any time and it will
```

Comment:
Will this component act as a global namespace similar to how `@workday/canvas-kit-react` imports all CKR components? Or would all of the unstable modules share and live in the same namespace `@workday/canvas-kit-react-unstable` and `@workday/canvas-kit-css-unstable`?

Reply:
Good point of discussion. I haven't built that into this yet, so I guess right now they're independently published. However, IMO all of the components (of the same target) should be exported under the same namespace. I'm not sure how we'd restrict them being published under their own namespace though. I think adding `private: true` to the package.json will do it.

Reply:
After further discussion, we arrived at a different conclusion due to a few things:

- We can't publish a bundle module while other ones are private if we plan to re-export them (because they won't be available as deps to download)
- Embedding them in the bundle module would be a pain and could lead to a lot of potential manual errors when migrating into unstable, or promoting out of unstable

In the end, we decided to organize these modules into the `_unstable` folder, but keep them as their own independent modules. They will be released with a warning in the readme and a prerelease version (i.e. if the codebase is on `3.0.0`, the menu module would be released as `3.0.0-beta`)